src/map.cpp

```diff
@@ -13,11 +13,15 @@
 
 namespace bpftrace {
 
-int Map::create_map(enum bpf_map_type map_type, const char *name, int key_size, int value_size, int max_entries, int flags) {
+int Map::create_map(enum bpf_map_type map_type, const std::string & name, int key_size, int value_size, int max_entries, int flags) {
+  std::string name_ = name;
```

Comment:
Style nit:
```suggestion
  std::string fixed_name = name;
```
We generally use trailing `_` for member variables, not locals.

Reply:
Done